pkgs/by-name/go/goframe-cli/package.nix

```
@@ -0,0 +1,51 @@
+{
+
+  lib,
+
+  buildGoModule,
+
+  fetchFromGitHub,
+
+}:
+
+buildGoModule rec {
+  pname = "gf";
+  version = "2.9.0";
+
+  src = fetchFromGitHub {
+    owner = "cococolanosugar";
+    repo = "gf";
+    rev = "v${version}";
+    hash = "sha256-o42G1dsz/WM1uI5kIsSZmLaBy9kHl4ZvT73KkB+hf7Y=";
+  };
+
+  subPackages = [ "." ];
+  modRoot = "cmd/gf";
+
+  vendorHash = null;
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  doCheck = false;
+
+  env = {
+    GOWORK = "off";
+  };
+
+  meta = with lib; {
+
+    description = "A powerful framework for faster, easier, and more efficient project development.";
+
+    homepage = "https://github.com/gogf/gf";
+
+    license = licenses.mit;
+
+    maintainers = with maintainers; [ "cococolanosugar" ];
```

```suggestion
    maintainers = with lib.maintainers; [ cococolanosugar ];
```
Maintainers must be declared in [`maintainers/maintainer.list.nix`](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix). Please follow the [Maintainers' README](https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md) to add your entry there.